gui-apps/swaync/swaync-0.7.3.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson vala gnome2-utils
+
+MY_PN="SwayNotificationCenter"
+DESCRIPTION="A simple notification daemon with a GTK gui for notifications and control center"
+HOMEPAGE="https://github.com/ErikReider/SwayNotificationCenter"
+SRC_URI="https://github.com/ErikReider/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+man scripting"
+
+DEPEND="
+	dev-libs/glib:2
+	dev-libs/gobject-introspection
+	dev-libs/json-glib
+	dev-libs/libgee:=
+	dev-libs/wayland
+	>=gui-libs/gtk-layer-shell-0.7.0[introspection]
+	gui-libs/libhandy:1
+	sys-apps/dbus
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:3
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	$(vala_depend)
+	man? ( app-text/scdoc )
+"
+
+src_prepare() {
+	default
+	vala_setup
+}
+
+src_configure() {
+	local emesonargs=(
+		$(meson_use man man-pages)
+		$(meson_use scripting)
```

Comment:
Upstream's README seems to suggest that it's built by default; is there any reason that this shouldn't be enabled unconditionally? Why would a user not want scripting support?

Reply:
You mean I should completely remove the USE flag? I agree that it probably should be enabled by default. Maybe some users don't need this feature and would prefer to build a more "minimal" version, so I figured they should be able to disable it.

Reply:
I'm thinking that; although choice is nice sometimes as a package maintainer you can make an executive decision.

In this case you could go either way - A user *might* reasonably want to disable scripting entirely, however it is given a section in the README and seems like a major feature. I don't use the software so I'll leave the decision entirely to you :)